File: CS/CS/E911.Module.Web/WebModuleEx.cs
using System;
using System.Linq;
using System.Collections.Generic;

namespace E911.Module.Web {
    public sealed partial class E911AspNetModule {
        // Registers custom extensions for Application Model elements.
        // Refer to the http://documentation.devexpress.com/#Xaf/CustomDocument3169 help article for more information.
        public override void ExtendModelInterfaces(DevExpress.ExpressApp.Model.ModelInterfaceExtenders extenders) {
            base.ExtendModelInterfaces(extenders);
            extenders.Add<E911.Module.Editors.IModelCustomUserControlViewItem, E911.Module.Web.Editors.IModelWebCustomUserControlViewItem>();
            extenders.Add<DevExpress.ExpressApp.SystemModule.IModelNavigationItem, E911.Module.Web.Controllers.IModelWebCustomFormPathNavigationItem>();
        }
    }
}

File: VB/CS/E911.Module.Web/WebModuleEx.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Linq
Imports System.Collections.Generic

Namespace E911.Module.Web
	Public NotInheritable Partial Class E911AspNetModule
		' Registers custom extensions for Application Model elements.
		' Refer to the http://documentation.devexpress.com/#Xaf/CustomDocument3169 help article for more information.
		Public Overrides Sub ExtendModelInterfaces(ByVal extenders As DevExpress.ExpressApp.Model.ModelInterfaceExtenders)
			MyBase.ExtendModelInterfaces(extenders)
			extenders.Add(Of E911.Module.Editors.IModelCustomUserControlViewItem, E911.Module.Web.Editors.IModelWebCustomUserControlViewItem)()
			extenders.Add(Of DevExpress.ExpressApp.SystemModule.IModelNavigationItem, E911.Module.Web.Controllers.IModelWebCustomFormPathNavigationItem)()
		End Sub
	End Class
End Namespace
